.github/ISSUE_TEMPLATE/1-Bug_report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Bug report
+about: You found a bug in expresslrs. 🐞
+labels: 'bug'
+---
+
+<!-- Please use the following issue template or your issue will be closed -->
+
+## Expected Behavior
```

Comment:
Remove this... we know the expected behavior. This is good for large projects or systems that have different states.